os/registry-authentication.md

```diff
@@ -0,0 +1,199 @@
+# Using authentication for a registry
+
+To pull images behind an authentication wall you need to setup some form of authentication with your image registry. These are either [quay.io][quay-site] or [hub.docker.com][docker-hub-site] or a self-hosted `registry.example.io/v0`.
+
+This document outlines the preferred method of authentication with an image registry like [quay.io][quay-site] and [hub.docker.com][docker-hub-site] and how setup registry auth for Docker, Kubernetes, rkt, and Mezos.
```

Comment:
@robszumski coreos.com docs have a table of contents on the sidebar. I can add another (manual) TOC if you think it'd be necessary. I can see pros for it. The main con is that it'd have to be hard-coded for it to show up on GitHub.